staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/validation_test.go

```diff
@@ -60,7 +57,6 @@ func TestValidateNestedValueValidationComplete(t *testing.T) {
 	for i := 0; i < tt.NumField(); i++ {
 		vv := &NestedValueValidation{}
 		x := reflect.ValueOf(&vv.ForbiddenExtensions).Elem()
-		i := rand.Intn(x.NumField())
```

Comment:
Removed this to make the test deterministic

Reply:
:+1: